README.md

```diff
@@ -14,7 +21,33 @@ yarn set version stable
 
 ### Windows
 
-_Add setup on windows steps here_
+### Package dependencies  
+
+- Install the Chocolatey package manager.
+- Run the following command in a Powershell 7 terminal:
+
+   ```pwsh
+   choco install chocolatey.config
```

Comment:
TODO: Update this to clarify that admin credentials are needed to run this command